source/apiVolontaria/volunteer/models.py

```diff
@@ -239,3 +244,12 @@ def __str__(self):
             self.standby,
             str(self.subscription_date),
         )
+
+    def delete(self, *args, **kwargs):
+        if not self.event.is_started:
+            super().delete(*args, **kwargs)
+        else:
+            raise BusinessException(
+                'You can\'t delete an application if the associated '
+                'event is already started'
```

Comment:
"You can't delete an application if the associated "
"event is already started."

I would stick with this convention: 
- Use single quotes for variable-like string
- Use double quotes for long strings/human message

See [this Stack Overflow answer](https://stackoverflow.com/a/56190) for a detailed explanation.